src/Analysis.cpp

```diff
@@ -538,8 +540,6 @@ void CodeAnalyzer::doAnalysis(Function &F, std::string optLoadFileName) {
   std::map<std::string, InstructionHolder> branchInstructionMap;
 
   for (inst_iterator I = inst_begin(F), E = inst_end(F); I != E; ++I) {
-    WorkSet.insert(&(*I));
```

Comment:
This change is not semantics-preserving (I think). Is it intentional? If so, what's the motivation for including it in this PR that otherwise is just style changes?

Reply:
WorkSet is unused